mix.exs

```diff
@@ -19,6 +20,11 @@ defmodule ExDoc.Mixfile do
     [clean: [&clean_test_fixtures/1, "clean"]]
   end
 
+  defp package do
+   [licenses: ["Apache 2.0"],
+    links: %{"GitHub" => "https://github.com/elixir-lang/ex_doc"}]
```

Comment:
@eksperimental No problem at all!, On the contrary, thanks for the feedback :+1: